web/beacon-app/src/features/projects/components/TopicTable.tsx

```diff
@@ -24,6 +25,10 @@ export const TopicTable = () => {
 
   console.log('topics data', topics); // do not remove this line. it is used for debugging since the topic id is not available in the UI
 
+  const redirectToTopicDetails = (topicID: string) => {
```

Comment:
Should an error be returned if for some reason the user isn't sent to the `Topic Detail Page`?